pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation.go

```diff
@@ -130,26 +130,6 @@ func initContainerName(lang language) string {
 	return fmt.Sprintf("datadog-lib-%s-init", lang)
 }
 
-func injectApmTelemetryConfig(pod *corev1.Pod) {
```

Comment:
Moved this into `libInfoSource.mutatePod()` since these both are telemetry metadata related and should be filtered to the same _container rules_.